src/components/EngineManageDialog.vue

```diff
@@ -261,7 +261,7 @@
             </div>
             <div class="no-wrap q-pl-md">
               <div class="text-h6 q-ma-sm">機能</div>
-              <ul v-if="engineManifests[selectedId]">
+              <ul v-if="engineManifests[selectedId].supportedFeatures">
```

Comment:
ここちょっとややこしいのですが、`engineManifests[selectedId]`がundefinedになる可能性があったりします。（未知のengineIdがあった場合など）
このとき`. supportedFeatures`がエラーになってしまって別の不具合になるかもです！

こういうときjavascriptは便利な気泡があって、こう書くと大丈夫なはずです。
```suggestion
              <ul v-if="engineManifests[selectedId]?.supportedFeatures">
```

Reply:
その記法、ESLintでしばってませんでしたっけ

Reply:
あ、そうでしたっけ。。。となるとこうですかね。。
```suggestion
              <ul v-if="engineManifests[selectedId] && engineManifests[selectedId].supportedFeatures">
```

Reply:
レビューありがとうございます。

この部分のループで使われる　selectedId　が engineIcons （≒engineManifests）から生成されるので
「構造自体は必ずあるが、supportedFeaturesはない可能性がある」という想定で書いてみました。

「その構造が崩れている状態は内部DBがすでにおかしいのでリカバリー不要」として
コードを削減する方向で記述してみましたが、ケアしておいたほうがよかったでしょうか？

Reply:
たしかに`engineManifests[selectedId]`は必ず存在しそうですね！
そのままでも特に問題なさそうに思いました！

（どちらかというと過去のコードの意図がわからなかったので、まあとりあえずundefinedの可能性があると想定したコードにしたほうが良いかな･･･。みたいな気持ちでした！）

Reply:
> selectedId　が engineIcons （≒engineManifests）から生成されるので

これは違いますね。（225行目のはアイコンを取れるかどうかの確認、アイコンのフォールバック用です）
エンジン管理ダイアログはエンジンが起動してなくても開ける必要があります。

Reply:
レビューありがとうございます。理解を間違えました。

selectedIdは変数で
・項目を選択したとき(74行目)に selectEngine() (559行目) でセットされる 
・初期値は　”” (421行目)

この挙動から行くと、状態が変化したときに
一時的にでも変数が一致しない状況は起きうるとおもったので、
レビュー頂いたコードを追加してPRします。